labs.yml

```diff
@@ -50,3 +50,8 @@ commands:
       - name: report
         default: all
         description: Report type
+  - name: validate-recon-config
+    description: Utility to validate Recon Config file.
+    flags:
+        - name: recon_conf
+          description: Path of the data-recon config
```

Comment:
shouldn't it be on the workspace?..

Reply:
this should be installed in user home when run through cli.

Reply:
and for validating if we don't find the config in user home then prompt for input path.

Reply:
User home on Databricks workspace?.. if not - it has to be on Databricks workspace, otherwise remorph is not coupled with Databricks and could be run without us. This is what we have to prevent with our OSS as much as possible.